examples/deadline/All-In-AWS-Infrastructure-Basic/ts/bin/config.ts

```diff
@@ -6,13 +6,23 @@
 import 'source-map-support/register';
 import { UsageBasedLicense } from 'aws-rfdk/deadline';
 import { MongoDbSsplLicenseAcceptance } from 'aws-rfdk';
+import { AwsThinkboxEulaAcceptance } from 'aws-rfdk/deadline';
 
 /**
  * Configuration values for the sample app.
  *
  * TODO: Fill these in with your own values.
  */
 class AppConfig {
+  /**
+   * Change this value to AwsThinkboxEulaAcceptance.USER_ACCEPTS_AWS_THINKBOX_EULA if you wish to accept the SSPL and
+   * proceed with Deadline deployment. Users must explicitly accept the AWS Thinkbox EULA before using the AWS Thinkbox
+   * Deadline container images.
+   *
+   * See https://www.awsthinkbox.com/end-user-license-agreement for the terms of the agreement.
+   */
+  public readonly acceptAwsThinkboxEula: AwsThinkboxEulaAcceptance = AwsThinkboxEulaAcceptance.USER_ACCEPTS_AWS_THINKBOX_EULA;
```

Comment:
Here should be `REJECTS` by default, I guess.

Reply:
Great catch! Must have accidentally committed my changes for testing